node_rate_loop: 20
width: 1600
height: 600
quality: 50
auto_exposure: true
exposure_value: 15
auto_wb: false
auto_wb_compensation: true
red_gain: 10
blue_gain: 50
binning: 2 # 2: no-binning, 1: x2-binning 0: x4-binning 3: x2 analog (special) - not working
binning_type: 1 # 0 :average, 1: sum

# Available Formats:
# mode: 7, 1600x600
# mode: 8, 2560x720
# mode: 9, 3840x1080
# mode: 10, 5184x1944
# mode: 11, 6528x1848
# mode: 12, 6528x2464